eclipse-cdt-cloud/cdt-cloud-blueprint
# fix: trigger Electron native build when packaging

As part of a merge of upstream electron-builder's native rebuilding was turned off. Upstream now uses Theia's Electron native rebuilding which was missed in CDT Cloud Blueprint and therefore lead to broken packaged builds. This is now fixed.

Fixes #60

Contributed on behalf of STMicroelectronics

#### How to test

Execute `git clean -dfx && yarn && yarn electron package` and verify that the built executable is launching instead of crashing.
